Sources_v3/WebSocketClient/EventMiddlewares/TypingStartCleanupMiddleware.swift

```diff
@@ -5,7 +5,7 @@
 import Foundation
 
 /// Automatically sends a `TypingStop` event if it hasn't come in a specified time after `TypingStart`.
-class UserTypingStartCleanupMiddleware<ExtraData: ExtraDataTypes>: EventMiddleware {
+class TypingStartCleanupMiddleware<ExtraData: ExtraDataTypes>: EventMiddleware {
```

Comment:
When on it, can you please add this middleware to the middlewares of `EventNotificationCenter`? It seems we have the implementation and tests, but we don't actually use it anywhere :) Be aware that `excludedUserIds` must reflect the current logged in user, which can change.